index.html

```diff
@@ -214,15 +214,14 @@
           </p>
           <p>
             LDP Containers MUST distinguish containment triples from membership and minimal-container triples.
-            If an implementation cannot make this distinction solely from the predicate, it MUST NOT permit
-            <code>ldp:contains</code> as the <code>membership predicate</code> for LDP Direct and Indirect
-            Containers or as a client-managed property, and requests that would do so MUST fail with 409 Conflict
-            and accompanying constraints document. Implementations that can distinguish containment triples MAY
-            permit  <code>ldp:contains</code> as a <code>membership predicate</code> or client-managed property;
-            such implementations SHOULD support <code>Prefer:</code> header values allowing clients to distinguish this data in the representation of a LDP-R as defined in [[!LDP]]:
-            <code>http://www.w3.org/ns/ldp#PreferContainment</code>,
-            <code>http://www.w3.org/ns/ldp#PreferMembership</code>, and
-            <code>http://www.w3.org/ns/ldp#PreferMinimalContainer</code>.
+            If an implementation cannot make this distinction, it MUST NOT permit <code>ldp:contains</code> as
+            the <code>membership predicate</code> for LDP Direct and Indirect Containers or as a client-managed
+            property, and requests that would do so MUST fail with 409 Conflict and accompanying constraints
+            document. Implementations that can distinguish containment triples MAY permit <code>ldp:contains</code>
+            as a <code>membership predicate</code> or client-managed property; such implementations SHOULD support
```

Comment:
This all looks good to me, with the exception of `such implementations SHOULD support...`.
It seems like these `Prefer` headers provide the mechanism for the user to distinguish containment from membership triples. 
Since much of the content of this PR is focused on being able to tease apart containment and membership triples, making that functionality available to users seems like a `MUST`.